Jenkinsfile

```diff
@@ -12,10 +12,6 @@ pipeline {
   }
 
   parameters {
-    booleanParam(name: 'skipGitHubSite',
-      description: 'If checked the plugin documentation on GitHub will NOT be updated',
-      defaultValue: true)
-

```

Comment:
I think this one of the reasons why the release pipeline tends to fail: ... we did not automatically run github sites creation (due to its slow runtime with egit). 
Now we will update the snapshot docs on every nightly, which is what we want I think